engine/grid.py

```diff
@@ -515,9 +552,10 @@ def get_waypoints(self, mode):
         elif mode == ControlMode.SPIRAL:
             waypoints = self.get_spiral_waypoints()
         elif mode == ControlMode.STRAIGHT:
-            waypoints = self.get_straight_line_waypoints(y_start_pct=0.5)
+            waypoints = self.get_straight_line_waypoints(y_start_pct =0.5)
         elif mode == ControlMode.LAWNMOWER_A:
             waypoints = self.get_all_lawnmower_waypoints_adjustable()
         else:
             return []
         return waypoints
+
```

Comment:
im putting this comment here since i cant add it to circle.png: there seem to be some sharp turns still - im assuming that the sock traversal isnt part of this PR? if it is not part of the PR, then nvm, ignore this comment!!

Reply:
It is not a part of this PR

Reply:
It is not a part of the PR